cluster/examples/kubernetes/ceph/toolbox.yaml

```diff
@@ -57,3 +57,8 @@ spec:
             items:
             - key: data
               path: mon-endpoints
+      tolerations:
```

Comment:
Why does the toolbox need this toleration? It's not critical

Reply:
neither harmful :-)